lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -253,9 +255,13 @@ function dosomething_kudos_term_is_selected($reportback_item, $term) {
     return false;
   }
 
-  if (! isset($reportback_item->existing_kids[$id])) {
+  if ((is_array($reportback_item) && empty($reportback_item['existing_kids'][$id])) && ! isset($reportback_item->existing_kids[$id])) {
     return false;
   }
 
   return true;
 }
+
```

Comment:
oh the comment chain didn't disappear since its on the line above the comment, silly :github: